scripts/rook.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 
-ROOK_VERSION=${ROOK_VERSION:-"v1.1.7"}
+ROOK_VERSION=${ROOK_VERSION:-"v1.2.7"}
```

Comment:
with this version, we will still get 14.2.7 which is not enough for cephfs.

@humblec you can patch the cluster.yaml https://github.com/rook/rook/blob/v1.2.7/cluster/examples/kubernetes/ceph/cluster-test.yaml#L17 to point to 14.2.10 which is required for cephfs E2E.